examples/stabilizer_code.py

```diff
@@ -0,0 +1,340 @@
+from typing import List, Tuple
+
+import numpy as np
+
+import cirq
+
+# Based on:
+# Stabilizer Codes and Quantum Error Correction
+# Daniel Gottesman
+# https://thesis.library.caltech.edu/2900/2/THESIS.pdf
+#
+# However, there were some mistakes in the thesis:
+# https://www2.perimeterinstitute.ca/personal/dgottesman/thesis-errata.html
```

Comment:
Standard form of a stabilizer code is also described in section 10.5.7. on page 470 in Nielsen & Chuang. I'm not sure whether any of the mistakes are corrected there, but they probably have given the 13 years that passed from the time the thesis was written and the latest N&C edition. Moreover, N&C may be a more familiar reference for most readers.

Reply:
[Add some background information] Yes, N&C book has description of standard form. But the error in the thesis is constructing encoding circuit from standard form stabilizer, which is not trivia at all. (You can tell that by Daniel himself made mistake in this. Twice.) The link Tony posted is the errata stated by Daniel, which is authentic. N&C's book doesn't describe the construction but leave it as problem 10.3 & 10.4. (Based on my own learning experience, it is not sufficient to list N&C book reference only here.) Maybe add more context in the comment is helpful. This is also good reason to introduce notebook since there are lots needed to be explained.

Reply:
Added a reference.